enhancements/ingress/set-delete-http-headers.md

```diff
@@ -732,32 +732,51 @@ We don't allow setting the `strict-transport-security` header on either an Ingre
 
-For example: If someone creates an Ingress Controller with the following:
-```sh
-apiVersion: operator.openshift.io/v1
-kind: IngressController
-metadata:
-  name: default
-  namespace: openshift-ingress-operator
-spec:
-  httpHeaders:
-    forwardedHeaderPolicy: Replace
-```
-Or if someone add a route annotation `haproxy.router.openshift.io/set-forwarded-headers`,
-then the `set-header` stanzas will be added in `haproxy.config` using the [haproxy-config.template](https://github.com/openshift/router/blob/0cd9a58dff77127562f7851ccfa5cda79b8cef48/images/router/haproxy/conf/haproxy-config.template#L565-L580)
+Note: In the IngressController, if both the action `Set` is configured using `spec.httpHeaders.actions` and the 
+action `Replace` is configured using `spec.httpHeaders.forwardedHeaderPolicy`, then the resulting configured value
+is from `spec.httpHeaders.actions`.  
+In short, the value set by `spec.httpHeaders.actions` will override the value set by `spec.httpHeaders.forwardedHeaderPolicy`.
+
+In the case of HTTP request headers, the actions specified in `spec.httpHeaders.actions` on the Route will be executed after
+the actions specified in the IngressController's `spec.httpHeaders.actions` field.
+
+This enhancement provides a method for a user to  delete the `x-forwarded-for` header, which the Ingress Controller `spec.httpHeaders.forwardedHeaderPolicy` does not provide.
+
+The `x-forwarded-for` header can be changed in three different places in the IngressControllerSpec and Route annotation: `spec.httpHeaders.actions`, `spec.httpHeaders.forwardedHeaderPolicy`, route annotation `haproxy.router.openshift.io/set-forwarded-headers`
+
+The following chart describes what values will be over-ridden when either `spec.httpHeaders.forwardedHeaderPolicy`, route annotation `haproxy.router.openshift.io/set-forwarded-headers` 
+or both are set along with `spec.httpHeaders.actions`.
+
+A=haproxy.router.openshift.io/set-forwarded-headers
 
-However, if someone wants to set custom values for these then they need to use the API of this EP.
+B=spec.httpHeaders.forwardedHeaderPolicy
 
-The custom values set using `spec.httpHeaders.actions` will take over the value defined in annotation `haproxy.router.openshift.io/set-forwarded-headers` or `spec.httpHeaders.forwardedHeaderPolicy`.  
+C=spec.httpHeaders.actions in IngressController
 
-Please note that the setting of forwardedHeaderPolicy happens at the plain backend section of haproxy.config i.e. per route level and not at the front-end section
-of the haproxy.config level as per the above-mentioned link.
+D=spec.httpHeaders.actions in Route
+
+| Simplified combination | When this comb. is present                                                                                                                                                  | This value is used                                                                                                               |
```

Comment:
Please remove the Simplied combination column, and the legend above.  Also, change:
```suggestion
| When these fields/annotation are set with a value |                                                                                                                                                | This value is used                                                                                                               |
```